master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -1,132 +1,63 @@
 ---
-title: Configuring calicoctl - etcdv2 datastore 
-layout: docwithnav
+title: Connecting calicoctl to etcd
 ---
 
-This document covers the configuration options for calicoctl when using an etcdv2 datastore.
-
-## Configuration file 
+## Configuration File
 
 By default `calicoctl` looks for a configuration file at `/etc/calico/calicoctl.cfg`.
 
 The file location may be overridden using the `--config` option on commands that required
 datastore access.
 
-The config file is a yaml map, with the following supported fields: 
```

Comment:
Not sure about the changes to this file... I don't think we should be getting rid of the reference table here + the example commands.  I think this was mostly fine as it was.